File: day_eleven/tempConversion.cpp
#include <iostream>

int main() {

    char unit;
    double userTemp, convertedTemp;

    std::cout << "Type F to enter in Fahrenheit or C to enter in Celcius:\n";
    std::cin >> unit;

    if (unit == 'F' || unit == 'f') {
        std::cout << "Enter the temperature in Fahrenheit:\n";
        std::cin >> userTemp;

        convertedTemp = (userTemp - 32.0) * (5.0/9.0);
        std::cout << convertedTemp << std::endl;
    }
    else if (unit == 'C' || unit == 'c') {
        std::cout << "Enter the temperature in Celcius:\n";
        std::cin >> userTemp;
        convertedTemp = (userTemp * ((9*1.0)/5) + 32);
        std::cout << convertedTemp << std::endl;
    }
    else {
        std::cout << "Invalid input\n";
    }


    return 0;
}

File: day_eleven/temp_conversion.py
#!/usr/bin/python3
def convert_temp():
    unit = input("Type F for Fahrenheit or C for Celcius: ")

    if unit.upper() == "F":
        temperature = float(input("Enter the temperature in fahrenheit:"))
        converted = (temperature - 32) * (5/9)
        print(converted)
    elif unit.upper() == "C":
        temperature = float(input("Enter the temperature in celcius: "))
        converted = (temperature * (9*1.0)/5.0) + 32
        print(converted)


def main():
    convert_temp()

if __name__=="__main__":
    main()